app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -582,6 +594,14 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
     }
   }
 
+  .checkbox-form-wrapper {
+    padding: 32px;
+
+    .checkbox-form {
+      padding-bottom: 24px;
```

Comment:
since we're using Crayons spacing units above, can we do that here?

According to `_import.scs`s, `24px` can mean `$su-6`

Reply:
Yes! I missed that the definitions for variables are in `_import.scss`. I'll pull from there and update :)

Reply:
@lisasy Is there any reason to not add a reference link to `_import.scss` to https://dev.to/page/crayons? I think it would have been helpful to know, and it seems good to add :)